plugins/cost.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cost
+spec:
+  version: v0.1.4
+  shortDescription: View cluster cost information
+  description: |
+    Uses Kubecost's API to gather and format cluster cost information broken
+    down by various aggregations like namespace, deployment, and label.
+
+    Kubecost is a cluster-side daemon that tracks the real cost of Kubernetes
+    resources by combining resource utilization monitoring with your provider's
+    cost data. Website: https://www.kubecost.com/
```

Comment:
It would be nice if you can list the supported cloud providers here. Or do you think this list is too fragile because you are adding more providers frequently?

Alternatively, you could also link to a list of supported providers. The goal is to make it easier for people to find out if this plugin applies to them.

Reply:
We have a fairly static list of supported providers. I don't expect it to change unless another cloud provider rockets to prominence in managed K8s. I've just added a list to the description.